index.d.ts

```diff
@@ -141,6 +141,11 @@ interface NavType {
     readerRevenueLinks: ReaderRevenuePositions;
 }
 
+interface NavBrowserType {
```

Comment:
Would it make more sense to call this `SubNavBrowserType` - calling it `NavBrowserType` confused me as I thought it was meant for `Nav`

Reply:
But thinking about it, it makes a lot more sense to not have a NavType anymore because we're no longer sending Nav done. We're just sending `currentNavLink` and `subNavSections` which could live by themselves?